### PR TITLE
[es] #1775 Issue: Localized the software-as-a-service.md document

### PR DESCRIPTION
**Pull request by ai-naymul**
### Describe your changes
Localized the content/en/software-as-a-service.md document into Spanish

### Related issue number or link (ex: `resolves #issue-number`)
Tracking issue: #1775 

### Checklist before opening this PR (put `x` in the checkboxes)
- [x] This PR does not contain plagiarism
  - don’t copy other people’s work unless you are quoting and contributing it to them.
- [x] I have signed off on all commits 
  - [signing off](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) (ex: `git commit -s`) is to affirm that commits comply [DCO](https://wiki.linuxfoundation.org/dco). If you are working locally, you could add an alias to your `gitconfig` by running `git config --global alias.ci "commit -s"`.

### CODE REVIEW

**Comment by ai-naymul** [comment]
@electrocucaracha I am a newbie contributor so If I have done any mistake please give me a review 
Thanks a lot for your consideration :)

**Comment by ramrodo** [comment]
Thanks for your contribution!

I just realized that this concept of _"Software as a Service (SaaS)"_ [is marked as Deprecated](https://github.com/cncf/glossary/blob/main/content/en/software-as-a-service.md?plain=1#L3) in the **main** branch.

Should we translate these concepts @electrocucaracha? I don't remember well.

**Comment by electrocucaracha** [comment]
> Thanks for your contribution!
> 
> I just realized that this concept of _"Software as a Service (SaaS)"_ [is marked as Deprecated](https://github.com/cncf/glossary/blob/main/content/en/software-as-a-service.md?plain=1#L3) in the **main** branch.
> 
> Should we translate these concepts @electrocucaracha? I don't remember well.

Right, maybe these terms should have the lowest priority until we discuss them with other maintainers.

**Comment by ai-naymul** [comment]
> Thanks for your contribution!
> 
> I just realized that this concept of _"Software as a Service (SaaS)"_ [is marked as Deprecated](https://github.com/cncf/glossary/blob/main/content/en/software-as-a-service.md?plain=1#L3) in the **main** branch.
> 
> Should we translate these concepts @electrocucaracha? I don't remember well.

Okay no problem😊

**Comment by krol3** [comment]
@ai-naymul  you need to change it to the branch `dev-es`,  instructions [here]([Contribution guide](https://github.com/cncf/glossary/blob/dev-es/content/es/contribute/_index.md#ayudar-a-traducir-el-glosario-help-translate-the-glossary))

**Comment by ai-naymul** [comment]
> @ai-naymul you need to change it to the branch `dev-es`, instructions [here]([Contribution guide](https://github.com/cncf/glossary/blob/dev-es/content/es/contribute/_index.md#ayudar-a-traducir-el-glosario-help-translate-the-glossary))

I have changed the branch to `dev-es` :)

File: content/es/software-as-a-service.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+---
+Title: Software as a Service (SaaS)
+Status: Deprecated
+Category: Technology
```
Comment (ramrodo) [inline content/es/software-as-a-service.md:4]: The category should be in Spanish

```suggestion
Category: Tecnología
```

**Review by ramrodo — CHANGES_REQUESTED** [review]
I'll like to invite you to review our [Contribution guide](https://github.com/cncf/glossary/blob/dev-es/content/es/contribute/_index.md#ayudar-a-traducir-el-glosario-help-translate-the-glossary), there are a few additional considerations for translating terms in Spanish.

File: content/es/software-as-a-service.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+SaaS proporciona soluciones de software completas mediante pago por uso.
+Todas las tareas de operación y mantenimiento, y los datos de la aplicación, son manejados por el proveedor de servicios.
+
+## Problema que aborda
+
+Traditionally, business software is installed on individual computers, requiring an admin to maintain and update.
+As an example: An organization may use on-premise software for customer relationship management (CRM).
+This software needs to be purchased, installed, secured, maintained, and regularly upgraded
+by the internal IT department, placing a burden on the IT team.
+The up front cost associated with licenses, installation, and potentially additional hardware can be prohibitive.
```
Comment (ramrodo) [inline content/es/software-as-a-service.md:22]: All these words are in English instead of Spanish.

**Review by ramrodo — CHANGES_REQUESTED** [review]

File: content/es/software-as-a-service.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
+---
+Title: Software como Servicio (SaaS)
+Status: Deprecated
+Category: Tecnología
+draft: true
+tags: ["fundamental", "platform", ""]
+---
+
+## Lo que es
```
Comment (ramrodo) [inline content/es/software-as-a-service.md:9]:
```suggestion
## ¿Qué es?
```

File: content/es/software-as-a-service.md [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+
+El software como servicio (SaaS) permite a los usuarios conectarse y utilizar servicios basados en la nube a través de Internet.
+Los ejemplos comunes son el correo electrónico, el calendario y las herramientas de oficina (como Gmail, Amazon Web Services, GitHub, Slack).
+SaaS proporciona soluciones de software completas mediante pago por uso.
+Todas las tareas de operación y mantenimiento, y los datos de la aplicación, son manejados por el proveedor de servicios.
+
+## Problema que aborda
```
Comment (ramrodo) [inline content/es/software-as-a-service.md:16]:
```suggestion
## ¿Cómo ayuda?
```

File: content/es/software-as-a-service.md [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+
+Tradicionalmente, el software comercial se instala en computadoras individuales, lo que requiere un administrador para mantenerlo y actualizarlo.
+Como ejemplo: una organización puede usar software local para la gestión de relaciones con los clientes (CRM).
+Este software debe comprarse, instalarse, protegerse, mantenerse y actualizarse regularmente
+por el departamento de TI interno, lo que supone una carga para el equipo de TI.
+El costo inicial asociado con las licencias, la instalación y el hardware potencialmente adicional puede ser prohibitivo.
+También puede ser difícil responder a la demanda y [escala](/es/scalability/) hacia arriba y hacia abajo
+según sea necesario rápidamente en respuesta al crecimiento o cambio.
+
+## Cómo ayuda
```
Comment (ramrodo) [inline content/es/software-as-a-service.md:26]:
```suggestion
## ¿Cómo ayuda?
```

**Comment by ramrodo** [comment]
Also, add the word(s) missing to the spellcheck file, please.

**Comment by ai-naymul** [comment]
> Also, add the word(s) missing to the spellcheck file, please.

Please check the file now
Thank you :)

**Comment by krol3** [comment]
Hi @ai-naymul , What is your local language, do you speak Spanish? Regards

**Comment by ai-naymul** [comment]
> Hi @ai-naymul , What is your local language, do you speak Spanish? Regards

No I don't speak spanish, our local language is Bengali and English.

**Review by ramrodo — APPROVED** [review]
I think the localization looks good to me now

**Review by electrocucaracha — APPROVED** [review]

**Pull request merged**